platform_unix.go

```diff
@@ -6,15 +6,40 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/coreos/go-systemd/v22/activation"
 	"net"
 )
 
 func platformFlags() {
 	flag.StringVar(&socket, "socket", "", "path to a Unix socket (e.g. /tmp/webhook.sock) to use instead of listening on an ip and port; if specified, the ip and port options are ignored")
+	flag.IntVar(&setGID, "setgid", 0, "set group ID after opening listening port; must be used with setuid, not permitted with -socket")
+	flag.IntVar(&setUID, "setuid", 0, "set user ID after opening listening port; must be used with setgid, not permitted with -socket")
```

Comment:
Since setuid and setgid don't work on Windows anyway, I've moved then in here so that they are only available as flags for non-Windows builds.